e2e/tests/test_cases/single_replica_node_down.robot

```diff
@@ -0,0 +1,135 @@
+*** Settings ***
+Documentation    Single replica node down
+...              https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-down/single-replica-node-down/
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/recurringjob.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/host.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources include off nodes
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Single Replica Node Down Deletion Policy do-nothing With RWO Volume Replica Locate On Replica Node
```

Comment:
These `Volume Replica Locate On Replica Node` type test cases are not mentioned in [the original manual test case](https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-down/single-replica-node-down/). Do you consider adding them and refining the manual one?

Reply:
Updated, manual test case refine PR [here](https://github.com/longhorn/longhorn-tests/pull/1913), verified agagin on local, thank you.